src/main/java/com/TeachMeSkills/jsp/hw/servlets/SaveRequestServlet.java

```diff
@@ -0,0 +1,17 @@
+package com.TeachMeSkills.jsp.hw.servlets;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet("/request")
+public class SaveRequestServlet extends HttpServlet {
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
```

Comment:
Why did you create separate servlet.
We could introduce this method in RegisteredServlet

Reply:
Updated: [e0c3ca42](https://github.com/Alexander-Khamitski/TMS_Java_EE/pull/4/commits/e0c3ca425f7c656b848e46d68534e274602c8430)